### ISSUE TITLE
The pcapi doesn't report the content-length of the assets

### ISSUE THREAD

**Issue by rgamez** [cobweb-eu/pcapi#30]
The pcapi doesn't return the header with the size of the assets, it is problematic with large files like the mbtiles see: edina/fieldtrip-cobweb-project#67

**Comment by rgamez** [comment]
```
curl -s -D - https://<endpoint>/1.3/pcapi/layers/local/<public-user-id>/test_3857.mbtiles -o /dev/null
```

```
HTTP/1.1 200 OK
Date: Mon, 23 Mar 2015 14:36:42 GMT
Server: Apache/2.4.7 (Ubuntu)
Access-Control-Allow-Origin: *
Access-Control-Allow-Methods: GET, POST, PUT, OPTIONS, DELETE
Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept
Vary: Accept-Encoding
Transfer-Encoding: chunked
Content-Type: text/html; charset=UTF-8
```

**Comment by rgamez** [comment]
Fixed here ec3c55e9767cdd52a7b751ee18aab7a793607381
